docs/index.md

```diff
@@ -0,0 +1,14 @@
+# Welcome to Metecho's documentation!
+
+```{toctree}
+:maxdepth: 2
+:caption: 'Contents:'
+
+api/index
+```
+
+# Indices and tables
+
+- {ref}`genindex`
+- {ref}`modindex`
```

Comment:
Not new in this PR, but this link seems to be broken -- I think because we're not using the `py:module::` directive? https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#directive-py-module